src/components/UI/Display/MAPI/clusters/GettingStarted/GettingStartedGetAccess.tsx

```diff
@@ -26,102 +27,74 @@ const GettingStartedGetAccess: React.FC<IGettingStartedGetAccessProps> = () => {
-                >
-                  from GitHub
-                </a>
-                , unpack the binary and move it to a location covered by your{' '}
-                <code>PATH</code> environment variable.
-              </Paragraph>
-            </>
-          }
-          windowsContent={
-            <>
-              <Paragraph fill={true}>
-                <a href='http://scoop.sh/' rel='noopener noreferrer'>
-                  scoop
-                </a>{' '}
-                enables convenient installs and updates for Windows PowerShell
-                users. Before you can install gsctl for the first time, execute
-                this:
-              </Paragraph>
-              <CodeBlock>
-                <Prompt>
-                  scoop bucket add giantswarm
-                  https://github.com/giantswarm/scoop-bucket.git
-                </Prompt>
-              </CodeBlock>
-              <Paragraph fill={true}>To install:</Paragraph>
-              <CodeBlock>
-                <Prompt>scoop install gsctl</Prompt>
-              </CodeBlock>
-              <Paragraph fill={true}>To update:</Paragraph>
-              <CodeBlock>
-                <Prompt>scoop update gsctl</Prompt>
-              </CodeBlock>
-              <Paragraph fill={true}>
-                To install without scoop, download the latest release{' '}
-                <a
-                  href='https://github.com/giantswarm/gsctl/releases'
-                  rel='noopener noreferrer'
-                >
-                  from GitHub
-                </a>
-                , unpack the binary and move it to a location covered by your{' '}
-                <code>PATH</code> environment variable.
-              </Paragraph>
-            </>
-          }
-        />
+        <CodeBlock>
+          <Prompt>kubectl krew install gs</Prompt>
+        </CodeBlock>
+        <Paragraph fill={true}>
+          Run this command to make sure the installation succeeded. You should
+          see information regarding the commands available:
+        </Paragraph>
+        <CodeBlock>
+          <Prompt>kubectl gs</Prompt>
+        </CodeBlock>
         <Paragraph fill={true}>
-          Run this command to make sure the installation succeeded:
+          To update <code>kubectl gs</code> to the latest version:
         </Paragraph>
         <CodeBlock>
-          <Prompt>{`gsctl --endpoint ${window.config.apiEndpoint} info`}</Prompt>
+          <Prompt>kubectl krew upgrade gs</Prompt>
         </CodeBlock>
         <Paragraph fill={true}>
-          Next, we let <code>gsctl</code> do several things in one step:
+          To install without Krew, download the{' '}
+          <a
+            href='https://github.com/giantswarm/kubectl-gs/releases/latest'
+            target='_blank'
+            rel='noopener noreferrer'
+          >
+            latest release
+          </a>{' '}
+          from GitHub for your platform, unpack the binary, and move it to a
```

Comment:
Not sure which wording is right here. When you're unpacking something, are you unpacking the box that it came in, or are you unpacking the contents of the box?

```suggestion
          from GitHub for your platform, unpack the archive, and move it to a
```